### ISSUE TITLE
Capframe Crash/Fatal Error when save in Libre Office

### ISSUE THREAD

**Issue by Scorpion101995** [CXWorld/CapFrameX#166]
I use Libre Office and when I save my File , CapframeX Crash every time with a fatal error (Report ID 753c6c65-44ce-4daa-9dc3-5854aca3257f )

**Comment by Taxxor90** [comment]
Ah so you're the one that was flooding our database with the same crashlog over and over again xD

The problem occurs when you rename your files while observing the folder in CX, but only if the folder is on a network drive.
This has been fixed internally for a couple of days now and will be available in the upcoming version 1.6.1. 
Until then, just don't rename your files while observing that directory

**Comment by Scorpion101995** [comment]
Yeah, pushing yes every time.  Already thought that would be a bit too often. :D

I don't use any network drive , so it is as you said the temporary files.

Thanks for the explanation.
